MacrobenchmarkSample/macrobenchmark/src/main/kotlin/com/example/macrobenchmark/baselineprofile/StartupProfileGenerator.kt

```diff
@@ -41,7 +42,9 @@ class StartupProfileGenerator {
             stableIterations = 3,
             includeInStartupProfile = true
         ) {
-            startActivityAndWait()
+            uiAutomator {
+                startApp()
```

Comment:
startActivityAndWait is a macrobenchmark function, this new one is a UIAutomator function. Are we sure this is correct? startActivityAndWait isn't doing anything else that now wouldn't be done?